kotlinlib/src/mill/kotlinlib/ksp/KspModule.scala

```diff
@@ -0,0 +1,157 @@
+package mill.kotlinlib.ksp
+
+import mill.*
+import mill.api.{PathRef, Result}
+import mill.define.Task
+import mill.kotlinlib.worker.api.{KotlinWorker, KotlinWorkerTarget}
+import mill.kotlinlib.{Dep, DepSyntax, KotlinModule}
+
+import java.io.File
+
+/**
+ * Sets up the kotlin compiler for using KSP (Kotlin Symbol Processing)
+ * by plugging in the symbol-processing and symbol-processing-api dependencies.
+ *
+ * Use of kotlin-compiler-embedded is also recommended (and thus enabled by default)
+ * to avoid any classpath conflicts between the compiler and user defined plugins!
+ */
+trait KspModule extends KotlinModule {
+
+  def symbolProcessingVersion: T[String]
```

Comment:
Can we name this `kspVersion` and add some docs?

Reply:
done, let me know if you'd like some different wording